federation/pkg/dnsprovider/providers/google/clouddns/internal/stubs/managed_zone_create_call.go

```diff
@@ -37,16 +37,16 @@ func (call ManagedZonesCreateCall) Do(opts ...googleapi.CallOption) (interfaces.
 	if call.Error != nil {
 		return nil, *call.Error
 	}
-	if call.Service.Impl[call.Project][call.ManagedZone.DnsName()] != nil {
+	if call.Service.Impl[call.Project][call.ManagedZone.Name()] != nil {
 		return nil, fmt.Errorf("Error - attempt to create duplicate zone %s in project %s.",
-			call.ManagedZone.DnsName(), call.Project)
+			call.ManagedZone.Name(), call.Project)
```

Comment:
In the original implementation ManagedZone.Name() and ManagedZone.DnsName() were set to be equal, so both work.  But I see that e.g. [ManagedZonesDeleteCall.Do()](https://github.com/kubernetes/kubernetes/blob/master/federation/pkg/dnsprovider/providers/google/clouddns/internal/stubs/managed_zones_delete_call.go#L44) uses ManagedZone.Name() as the key, so this change improves code consistency.
From https://godoc.org/google.golang.org/api/dns/v1#ManagedZone 

```
// DnsName: The DNS name of this managed zone, for instance
// "example.com.".

// Name: User assigned name for this resource. Must be unique within the
// project. The name must be 1-63 characters long, must begin with a
// letter, end with a letter or digit, and only contain lowercase
// letters, digits or dashes.